src/js/domain/models/Lotto/Lotto.js

```diff
@@ -0,0 +1,63 @@
+import {
+    LottoNumbersNotArrayError,
+    LottoNumbersLengthNotSixError,
+    LottoNumberNotNumberError,
+    LottoNumberOutOfRangeError,
+    LottoNumberDuplicatedError,
+} from './errors.js';
+import {
+    LOTTO_DIGITS,
+    LOTTO_LOWER_BOUND,
+    LOTTO_UPPER_BOUND,
+} from '../../constants.js';
+
+export default class Lotto {
+    #lottoNumbers;
+
+    static of(lottoNumbers) {
+        return new Lotto(lottoNumbers);
+    }
+
+    constructor(lottoNumbers) {
+        this.#validateLottoNumbers(lottoNumbers);
+
+        this.#lottoNumbers = lottoNumbers;
```

Comment:
```suggestion
        this.#lottoNumbers = new Set(lottoNumbers);
```
이렇게 아예 set으로 만들어서 관리해주면 어떨까요~?
값 자체가 중복되지 않는다는 것을 자료구조를 통해서 증명할 수 있지 않을까 싶어요!

Reply:
💡 네! 피드백 주신 것처럼 `Map`, `Set`, `WeakMap`, `WeakSet` 같은 **자바스크립트에서 지원하는 빌트인 객체를 활용해서 보다 해당 데이터의 구조적인 특성을 드러내면 좋을 것** 같아요. 

반영해두겠습니다.